.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```

Comment:
I'm curious why this change is necessary. It seems unrelated to fixing code coverage reporting.